includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -611,6 +562,76 @@ private function getFollowUs() {
 		return $data;
 	}
 
+	private function getCommunity() {
+		$data = [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-footer-wikia-community-header'
+				]
+			],
+			'links' => [ ]
+		];
+
+		if ( $this->getHref( 'community-central' ) ) {
+			$data['links'][] = [
```

Comment:
That's why I created the upgraded fix version in the PR linked in description—https://github.com/Wikia/app/pull/10961. :)